source/_integrations/ezviz.markdown

```diff
@@ -28,6 +28,14 @@ As there is no official documentation for the API, the integration retrieves dat
 
 The password for each camera is usually written near the QR code. This could be underneath the device or in the user manual. It is usually referred to as the camera "verification code".
 
+The local RTSP server on your camera [needs to be enabled](https://community.home-assistant.io/t/ezviz-camera-discovered-but-unable-to-connect/535885). To do that:
```

Comment:
What does the community link add here? I think we can remove it?

Reply:
Yes, it's just a source of the information.